source/unified-test-format/tests/valid-pass/poc-queryable-encryption.yml

```diff
@@ -0,0 +1,84 @@
+description: poc-queryable-encryption
+
+schemaVersion: "1.23"
+
+runOnRequirements:
+  - minServerVersion: "7.0"
+     csfle: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+      autoEncryptOpts:
+        kmsProviders:
+          local:
+            key: Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name poc-queryable-encryption
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+  - database:
+      id: &database1 database1
+      client: *client0
+      databaseName: &database1Name keyvault
+  - collection:
+      id: &collection1 collection1
+      database: *database0
+      collectionName: &collection1Name datakeys
+
+
+initialData:
+  - databaseName: *database1Name
+    collectionName: *collection1Name
+    documents:
+      - _id: &keyid { $binary: { base64: EjRWeBI0mHYSNBI0VniQEg==, subType: "04" } }
+        keyMaterial: { $binary: { base64: sHe0kz57YW7v8g9VP9sf/+K1ex4JqKc5rf/URX3n3p8XdZ6+15uXPaSayC6adWbNxkFskuMCOifDoTT+rkqMtFkDclOy884RuGGtUysq3X7zkAWYTKi8QAfKkajvVbZl2y23UqgVasdQu3OVBQCrH/xY00nNAs/52e958nVjBuzQkSb1T8pKJAyjZsHJ60+FtnfafDZSTAIBJYn7UWBCwQ==, subType: "00" } }
+        creationDate: { $date: { $numberLong: "1641024000000" } }
+        updateDate: { $date: { $numberLong: "1641024000000" } }
+        status: 1
+        masterKey:
+          provider: local
+  - databaseName: *database0Name
+    collectionName: *collection0Name
+    documents: []
+    createOptions:
+      encryptedFields:
+        fields:
+          - keyId: *keyid
+            path: 'encryptedInt'
+            bsonType: 'int'
+            queries: {'queryType': 'equality', 'contention': {'$numberLong': '0'}}
+
+tests:
+  - description: insert, replace, and find with queryable encryption
+    operations:
+      - object: *collection0
+        name: insertOne
+        arguments:
+          document:
+            _id: 1
+            encryptedInt: 11
+      - object: *collection0
+        name: replaceOne
+        arguments:
+          filter: { encryptedInt: 11 }
+          replacement: { encryptedInt: 22 }
+      - object: *collection0
+        name: find
+        arguments:
+          filter: { encryptedInt: 22 }
+        expectResult:
+          - _id: 1
+            encryptedInt: 22
+    outcome:
+      - collectionName: *collection0Name
+        databaseName: *database0Name
+        documents:
+          - { _id: 1, encryptedInt: { $$type: binData }, __safeContent__: { $$type: array} }
```

Comment:
I saw this line in the PR description:

> Note: the server forbids inserting documents with __safeContent__ fields. Therefore the  initialData entity does not support inserting ciphertext for QE. Use insert commands with plaintext instead.

Is there actually a need for "inserting ciphertext for QE"? I don't see an `insert` command being used anywhere.

The [CSFLE test README](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.md) doesn't talk about this, and AFAIK `__safeContent__` only shows up in `outcome` blocks within the [legacy tests](https://github.com/mongodb/specifications/tree/master/source/client-side-encryption/tests/legacy).